src/test/java/com/java_academy/logic/model/PlayerTest.java

```diff
@@ -0,0 +1,48 @@
+package com.java_academy.logic.model;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class PlayerTest {
+	
+	private Player player;
+	private int nukeCounter = 3;
+
+	@BeforeTest
+	public void createPlayer() {
+		player = new Player("first", nukeCounter);
+		player.createFleet();
+	}
+	
+	@Test
+	public void testPlayerCanUseNuke() {
```

Comment:
Think about change name convention. for example 
shouldReturnTrueIfPlayerCanUseNuke